src/components/Form/Form.stories.tsx

```diff
@@ -494,7 +636,7 @@ export const Playground = (): ReactElement => {
 					state={state}
 					fields={fields}
 					dispatch={dispatch}
-					getFormValues={loadReady && getFormValues}
+					getFormValues={showGetFormValues === "None" ? loadReady && undefined : loadReady && getFormValues}

```

Comment:
If showGetFormValues is "None" that means getFormValues is undefined, so we don't actually care if loadReady is true or not. I'd also suggest adding the parentheses to make it easier to read.
```suggestion
					getFormValues={showGetFormValues === "None" ? undefined : (loadReady && getFormValues)}

```